crates/router/src/core/payouts.rs

```diff
@@ -2733,6 +2750,28 @@ pub async fn make_payout_data(
         payouts::PayoutRequest::PayoutRetrieveRequest(_) => None,
     };
 
+    if let Some(payout_method_data) = payout_method_data_req.clone() {
+        let additional_payout_method_data =
+            helpers::get_additional_payout_data(&payout_method_data, &*state.store, &profile_id)
+                .await;
+
+        let update_additional_payout_method_data =
+            storage::PayoutAttemptUpdate::AdditionalPayoutMethodDataUpdate {
+                additional_payout_method_data,
+            };
+
+        payout_attempt = db
+            .update_payout_attempt(
+                &payout_attempt,
+                update_additional_payout_method_data,
+                &payouts,
+                merchant_account.storage_scheme,
+            )
+            .await
+            .change_context(errors::ApiErrorResponse::InternalServerError)
+            .attach_printable("Error updating routing info in payout_attempt")?;
```

Comment:
Should this have been "additional payout method data" instead of "routing info"?